client/executor/wasmi/src/lib.rs

```diff
@@ -536,6 +577,10 @@ pub struct WasmiRuntime {
 	state_snapshot: StateSnapshot,
 	/// The host functions registered for this instance.
 	host_functions: Vec<&'static dyn Function>,
+	/// Enable STUB for function called that are missing
+	enable_stub: bool,
```

Comment:
Instead of `enable_stub` I'd like to suggest to use something like `allow_missing_imports`. The rationale is that in the wasm spec a missing import is specified an instantiation error, so `allow_missing_imports` would convey that this is a custom mode. `enable_stub` is on contrary a little bit vague since there might be different kinds of stubs out there.

Aside, expanding the role of this field in the doc would be great!

Reply:
Fine by me. @bkchr any remark?

Reply:
I like @pepyakin idea :) Sounds much better!

Reply:
done